scripts/release.sh

```diff
@@ -383,6 +383,10 @@ function main() {
     abort "error building the release"
   fi
   [[ -z "${YAMLS_TO_PUBLISH}" ]] && abort "no manifests were generated"
+  # Ensure no empty YAML file will be published.
```

Comment:
Should this be here or few lines below under `if (( PUBLISH_RELEASE)); then` ?

Reply:
Here. An empty yaml is a build error.